app/controllers/tool_cart_controller.rb

```diff
@@ -62,6 +62,18 @@ def checkout
                     locals: {message: "Invalid organization"}
     end
 
+    participant_certs = participant.certifications.map { |cert| cert.certification_type.name }
```

Comment:
Unfortunately this can't be done because of the current tool checkout workflow, since when we add tools to the cart, we don't know what participant will be checking the tools out. It is only in the checkout when we know the participant that is trying to checkout tools.